data/postgres/00_domestic_control.sql

```diff
@@ -1,99 +1,92 @@
-DROP TABLE IF EXISTS rooms CASCADE;
-CREATE TABLE rooms (
+CREATE SCHEMA IF NOT EXISTS domestic;
+
+DROP TABLE IF EXISTS domestic.rooms CASCADE;
+CREATE TABLE domestic.rooms (
   "id" TEXT PRIMARY KEY,
   "name" TEXT,
   "group" TEXT
 );
 
--- DROP TYPE IF EXISTS control_type CASCADE;
--- CREATE TYPE control_type AS ENUM ('light', 'blind', 'amplifier', 'temperature', 'humidity', 'co2');
-
-DROP TABLE IF EXISTS controls CASCADE;
-CREATE TABLE controls (
+DROP TABLE IF EXISTS domestic.conditions CASCADE;
+CREATE TABLE domestic.conditions (
   "id" TEXT PRIMARY KEY,
-  "room_id" VARCHAR REFERENCES rooms("id"),
+  "room_id" VARCHAR REFERENCES domestic.rooms("id"),
   "type" TEXT,
   "name" TEXT
 );
 
--- DROP TYPE IF EXISTS sensor_type CASCADE;
--- CREATE TYPE sensor_type AS ENUM ('temperature', 'humidity', 'co2');
-
-DROP TABLE IF EXISTS sensors CASCADE;
-CREATE TABLE sensors (
+DROP TABLE IF EXISTS domestic.blinds CASCADE;
+CREATE TABLE domestic.blinds (
   "id" TEXT PRIMARY KEY,
-  "room_id" VARCHAR REFERENCES rooms("id"),
-  "type" TEXT,
-  "name" TEXT
-);
-
--- DROP TYPE IF EXISTS blinds_opacity CASCADE;
--- CREATE TYPE blinds_opacity AS ENUM ('shear', 'blind');
-
-DROP TABLE IF EXISTS blinds CASCADE;
-CREATE TABLE blinds (
-  "id" TEXT PRIMARY KEY,
-  "room_id" VARCHAR REFERENCES rooms("id"),
+  "room_id" VARCHAR REFERENCES domestic.rooms("id"),
   "name" TEXT,
   "opacity" TEXT,
   "group" TEXT
 );
 
-DROP TABLE IF EXISTS lighting_groups CASCADE;
-CREATE TABLE lighting_groups (
+DROP TABLE IF EXISTS domestic.lighting_groups CASCADE;
+CREATE TABLE domestic.lighting_groups (
   "id" VARCHAR PRIMARY KEY,
-  "room_id" VARCHAR REFERENCES rooms("id"),
+  "room_id" VARCHAR REFERENCES domestic.rooms("id"),
   "name" TEXT
 );
 
-DROP TABLE IF EXISTS amplifiers CASCADE;
-CREATE TABLE amplifiers (
+DROP TABLE IF EXISTS domestic.amplifiers CASCADE;
+CREATE TABLE domestic.amplifiers (
   "id" VARCHAR PRIMARY KEY,
-  "room_id" VARCHAR REFERENCES rooms("id"),
+  "room_id" VARCHAR REFERENCES domestic.rooms("id"),
   "name" TEXT
 );
 
-DROP TABLE IF EXISTS rooms_controls_log CASCADE;
-CREATE TABLE rooms_controls_log (
-  "id" VARCHAR REFERENCES controls("id"),
-  "room_id" VARCHAR REFERENCES rooms("id"),
+DROP TABLE IF EXISTS domestic.room_controls CASCADE;
+CREATE TABLE domestic.room_controls (
+  "id" TEXT PRIMARY KEY,
+  "room_id" VARCHAR REFERENCES domestic.rooms("id"),
   "name" TEXT,
   "type" TEXT,
-  "value" REAL,
+  "value" TEXT,
   "time" TIMESTAMPTZ
 );
 
-DROP TABLE IF EXISTS rooms_sensors_log CASCADE;
-CREATE TABLE rooms_sensors_log (
-  "id" VARCHAR REFERENCES controls("id"),
-  "room_id" VARCHAR REFERENCES rooms("id"),
+DROP TABLE IF EXISTS domestic.room_controls_log CASCADE;
+CREATE TABLE domestic.room_controls_log (
+  "id" VARCHAR,
```

Comment:
Maybe comment on why this `id` isn't a PRIMARY KEY (or rename it to something else)